app/javascript/packs/ping_thing/index_template.vue

```diff
@@ -4,45 +4,80 @@
 
     <div class="card mb-4">
       <div class="card-content">
-        <h3 class="card-heading mb-4">{{ network[0].toUpperCase() + network.substring(1) }} TX Confirmation Time</h3>
-        <scatter-chart :vector="ping_things.slice().reverse()" fill_color="rgba(221, 154, 229, 0.4)" line_color="rgb(221, 154, 229)"/>
+        <h3 class="card-heading">{{ network[0].toUpperCase() + network.substring(1) }} TX Confirmation Time</h3>
+        <div class="text-center pb-4">
+        <b-pagination
+            v-model="page"
+            :total-rows="total_count"
+            :per-page="240"
+            first-text="« First"
+            last-text="Last »" />
+        </div>
+        <bubble-chart :vector="ping_things.slice().reverse()" fill_color="rgba(221, 154, 229, 0.4)" line_color="rgb(221, 154, 229)"/>
       </div>
     </div>
 
     <div class="card">
-      <ping-thing-table :ping_things="ping_things"/>
+      <ping-thing-table :ping_things="ping_things" />
+      <b-pagination
+          v-model="page"
+          :total-rows="total_count"
+          :per-page="240"
+          first-text="« First"
+          last-text="Last »" />
     </div>
   </div>
 </template>
 
 <script>
   import axios from 'axios'
-  import scatterChart from './scatter_chart'
+  import bubbleChart from './bubble_chart'
   import pingThingHeader from './ping_thing_header'
   import pingThingTable from './ping_thing_table'
-  import Ping_thing_table from './ping_thing_table.vue'
 
   axios.defaults.headers.get["Authorization"] = window.api_authorization
 
   export default {
     props: ['network'],
     data () {
+      var api_url = '/api/v1/ping-thing/' + this.network + '?with_total_count=true'
       return {
-        ping_things: []
+        ping_things: [],
+        page: 1,
+        total_count: 0,
+        api_url: api_url
       }
     },
     created () {
       var ctx = this
-      axios.get("/api/v1/ping-thing/" + this.network)
+      axios.get(ctx.api_url)
```

Comment:
perhaps using axios params would be a better idea? 
`axios.get(ctx.api_url, { params: { with_total_count: true } })`